app/src/main/java/com/kickstarter/viewmodels/MessagesViewModel.java

```diff
@@ -39,14 +46,20 @@
     /** Emits a boolean that determines if the backing info view should be hidden. */
     Observable<Boolean> backingInfoViewHidden();
 
-    /** Emits the participant name to be displayed. */
-    Observable<String> participantNameTextViewText();
-
     /** Emits a list of messages to be displayed. */
     Observable<List<Message>> messages();
 
+    /** Emits the participant name to be displayed. */
+    Observable<String> participantNameTextViewText();
+
     /** Emits the project name to be displayed. */
     Observable<String> projectNameTextViewText();
+
+    /** Emits an empty string when we want to clear the message edit text. */
+    Observable<String> setEmptyMessageEditText();
+
+    /** Emits when we should show a message error toast. */
+    Observable<Boolean> showMessageErrorToast();
```

Comment:
makes sense! this Boolean is doing nothing and it should be a string

this `Boolean` value comes from an initial iteration where I had hoped we could get around using a `PublishSubject` as an output for toasts (since we don't want these errors to show again and again on rotation) but doing so required more logic around the output vs. just using a publish subject for now to get that behavior for free ¯\_(ツ)_/¯